README.rst

```diff
@@ -119,6 +119,12 @@ Added
 - Support ``Nexmo`` as a backend service alongwith ``Twilio``.
 - Add docs for writing custom backend.
 
+
+Changed
+"""""""
+- Update ``backends.base.validate_security_code`` to use ``save()`` instead of ``update()`` to allow Django to emit its ``post_save()`` signal.
```

Comment:
```suggestion
- Update ``backends.base.BaseBackend.validate_security_code`` to use ``save()`` instead of ``update()`` to allow Django to emit its ``post_save()`` signal.
```